hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -546,6 +551,9 @@ public DeltaSyncService(Config cfg, JavaSparkContext jssc, FileSystem fs, Config
         cfg.baseFileFormat = meta.getTableConfig().getBaseFileFormat().toString();
         this.cfg.baseFileFormat = cfg.baseFileFormat;
       } else {
+        if (fs.exists(new Path(cfg.targetBasePath))) {
+          fs.delete(new Path(cfg.targetBasePath));
```

Comment:
This API has been deprecated. This operation could fail as well. Maybe we can add some exception message here.

Reply:
ok